modules/ingestor/src/service/advisory/osv/loader.rs

```diff
@@ -173,6 +173,26 @@ impl<'g> OsvLoader<'g> {
                                 )
                                 .await?;
                             }
+                            (RangeType::Ecosystem, Ecosystem::PyPI) => {
```

Comment:
Maybe this can be a single match?

`(RangeType::Ecosystem, Ecosystem::PyPI | Ecosystem::Python) => {`

Reply:
Absolutely, you're right: at least I learnt something new about Rust also today 😉 thanks